charts/umbrella/Chart.yaml

```diff
@@ -28,7 +28,7 @@ sources:
   - https://github.com/eclipse-tractusx/tractus-x-umbrella
 
 type: application
-version: 2.0.3
+version: 2.0.4
```

Comment:
hmm just noticed 2.0.4 is already the latest version, not sure why it's not showing a conflict, could you please raise to 2.0.5

Reply:
Thanks for the hin. I updated it to version 0.2.5